tracing-flame/src/lib.rs

```diff
@@ -158,9 +158,7 @@ use tracing_subscriber::Subscribe;
 
 mod error;
 
-lazy_static! {
-    static ref START: Instant = Instant::now();
-}
+static START: Lazy<Instant> = Lazy::new(|| Instant::now());
```

Comment:
this emits a Clippy warning that's breaking CI: https://github.com/tokio-rs/tracing/runs/6759776017?check_suite_focus=true

i think we should change this to 
```suggestion
static START: Lazy<Instant> = Lazy::new(Instant::now);
```
to fix that warning.